config/newrelic.yml

```diff
@@ -0,0 +1,45 @@
+#
+# This file configures the New Relic Agent.  New Relic monitors Ruby, Java,
+# .NET, PHP, Python and Node applications with deep visibility and low
+# overhead.  For more information, visit www.newrelic.com.
+#
+# Generated October 28, 2017
+#
+# This configuration file is custom generated for SafecastAPI
+#
+# For full documentation of agent configuration options, please refer to
+# https://docs.newrelic.com/docs/agents/ruby-agent/installation-configuration/ruby-agent-configuration
+
+common: &default_settings
+  # Required license key associated with your New Relic account.
+  license_key: <%= ENV['NEWRELIC_LICENSE_KEY'] %>
+
+  # Your application name. Renaming here affects where data displays in New
+  # Relic.  For more details, see https://docs.newrelic.com/docs/apm/new-relic-apm/maintenance/renaming-applications
+  app_name: ingest
+
+  # To disable the agent regardless of other settings, uncomment the following:
+  # agent_enabled: false
```

Comment:
Just noticed this option exists. Thinking we might want to interpolate here that way we get an easy true/false toggle when we want to put it into APM.

Reply:
Nice. Enabled this option, and `false` by default and `true` in production.